openapi/components/schemas/Profile.yaml

```
@@ -60,6 +76,14 @@ properties:
     type: string
     format: url
     deprecated: true
+  status:
+    description: Status of the user's account.
```

I think here `'s` is not needed, or mistaken @noon-dawg?

```suggestion
    description: Status of the user account.
```